tofu/geom/_def_config.py

```diff
@@ -169,11 +169,11 @@
         'CoilPF': ['PF1', 'PF2', 'PF3', 'PF4', 'PF5', 'PF6'],
         'CoilCS': ['CS3U', 'CS2U', 'CS1U', 'CS1L', 'CS2L', 'CS3L'],
     },
-    'ITER-SOLEDGE3XV0': {
-        'Exp': _ExpITER,
-        'Ves': ['SOLEDGE3XV0'],
-        'PFC': ['SOLEDGE3XDivDomeV0', 'SOLEDGE3XDivSupportV0']
-    },
+    # 'ITER-SOLEDGE3XV0': {
+    # 'Exp': _ExpITER,
+    # 'Ves': ['SOLEDGE3XV0'],
+    # 'PFC': ['SOLEDGE3XDivDomeV0', 'SOLEDGE3XDivSupportV0']
+    # },
```

Comment:
why not just erase these lines?

Reply:
In case we need to create it again in the future, it's just a precaution.